### DISCUSSION TITLE
Pull Request for all Angular 11 versions.

### DISCUSSION THREAD

**Discussion by nbaker82** [tanansatpal/ngx-mat-intl-tel-input#100] [General]
The current version of this library only works for the version in the package.json file and the peerDependency's were never updated. I just put out a pullRequest that updates those. If you could take a look at it that would be great.  We really like using this library and would like for it to work in future versions of angular.

**Comment by danj1980** [comment]
Any thoughts as to whether this project is being actively managed?

**Reply by nbaker82** [reply]
I don't think it is anymore.  We need to update angular on some projects and this library is preventing us from updating.  So I may just fork it coming up.

**Reply by AdGalesso** [reply]
Yeah had to the same
